ffmpeg/videoprofile.go

```diff
@@ -246,17 +244,23 @@ func ParseProfiles(injson []byte) ([]VideoProfile, error) {
 			} else {
 				gopFloat, err := strconv.ParseFloat(profile.GOP, 64)
 				if err != nil {
-					return parsedProfiles, fmt.Errorf("Cannot parse the GOP value in the transcoding options: %w", err)
+					return parsedProfiles, fmt.Errorf("cannot parse the GOP value in the transcoding options: %w", err)
 				}
-				if gopFloat <= 0.0 {
-					return parsedProfiles, fmt.Errorf("Invalid gop value %f. Please set it to a positive value", gopFloat)
+				// gopFloat is allowed to be 0
+				if gopFloat < 0.0 {
```

Comment:
Tests in `go-livepeer` repo are designed to test for `0` in `profile.GOP`. Fixing this condition to allow 0